src/main/java/usecases/file_management/FileManagementOutputBoundary.java

```
@@ -0,0 +1,11 @@
+package usecases.file_management;
+
+public interface FileManagementOutputBoundary {
+    default void prepareFileListView(FileManagementOutputData outputData) {
```

Using default implementations in an interface defeats the purpose of the interface. If these methods have default (empty) implementations, then the implementing class doesn't have to implement them, which could lead to silent failures.

Better to make these abstract methods (no default implementation) so that any implementing class is forced to provide real implementations.